src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/BalanceSheetServiceImpl.java

```diff
@@ -93,10 +107,12 @@ private CurrentPeriodApi getCurrentPeriod(ApiClient apiClient, UriTemplate uri,
         }
     }
 
-    private PreviousPeriodApi getPreviousPeriod(ApiClient apiClient, UriTemplate uri, String transactionId, String companyAccountsId) throws ServiceException {
+    private PreviousPeriodApi getPreviousPeriod(ApiClient apiClient, UriTemplate uri,
```

Comment:
We don't need to pass `UriTemplate uri` to this private method it is always the `PREVIOUS_PERIOD_URI` which is available to the whole class.